app/models/distribution.rb

```diff
@@ -154,9 +154,7 @@ def reset_shipping_cost
     self.shipping_cost = nil unless delivery_method == "shipped"
   end
 
-  def validate_shipping_cost
-    if shipping_cost&.negative?
-      raise StandardError.new("Shipping cost cannot be negative!")
-    end
+  def shipped?
```

Comment:
I don't think this method needs to be defined? It should be defined automatically when you have an enum. https://api.rubyonrails.org/v7.0/classes/ActiveRecord/Enum.html